homeassistant/components/minecraft_server/quality_scale.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration doesn't provide any service actions.
+  appropriate-polling:
+    status: done
+    comment: Default polling interval is set to 60s.
+  brands: done
+  common-modules: done
+  config-flow: done
+  config-flow-test-coverage: done
```

Comment:
`test_show_config_form` doesn't really do anything and can be merged with the full flow test. (Also, big preference if we move the full flow test to the top of the config flow tests)